redaxo/src/addons/be_style/plugins/redaxo/assets/icons/browserconfig.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
+        <tile>
+            <square150x150logo src="/assets/addons/be_style/plugins/redaxo/icons/mstile-150x150.png"/>
```

Comment:
REDAXO kann auch in Unterordnern installiert werden. Wie handhaben wir `$this->getAssetsUrl` in statischen Dateien?

Reply:
Achso ja, stimmt. ;)
Okay, dann müssen wir in den beiden Files wohl search-&-replacen. Ich schaue mal, ob ich das heute abend rein kriege. Ansonsten gerne wie immer: Help wanted.

Reply:
@tbaddade Du hast recht. Habe gerade gelesen, relative Pfade sind laut Specs sowohl in `browserconfig.xml` als auch in `manifest.json` möglich. Passe ich an…